tests/unit/core/reducers/test_addons.js

```diff
@@ -566,6 +593,15 @@ describe(__filename, () => {
         createInternalAddon(fakeAddon),
       );
     });
+
+    it('is case insensitive', () => {
+      const { store } = dispatchClientMetadata();
+      store.dispatch(loadAddons(createFetchAddonResult(fakeAddon).entities));
+
+      expect(
+        getAddonBySlug(store.getState(), fakeAddon.slug.toUpperCase()),
```

Comment:
Nit: As above, we're assuming that `fakeAddon.slug` is not already in upper case.

Reply:
I'd argue that the test case above explicitly proves that.

Reply:
I'd argue that that statement proves my case. ;-)

Reply:
haha, I changed it anyway